apps/filebrowser/src/filebrowser/views.py

```diff
@@ -936,18 +954,27 @@ def _read_simple(fhandle, path, offset, length, stats):
 
 def detect_gzip(contents):
   '''This is a silly small function which checks to see if the file is Gzip'''
-  return contents[:2] == '\x1f\x8b'
+  if sys.version_info[0] > 2:
+    return contents[:2] == b'\x1f\x8b'
+  else:
+    return contents[:2] == '\x1f\x8b' 
 
```

Comment:
Here was thinking more one liners to avoid the duplications, particular above. But one advantage is that it will be easier to remove the Python2 code later on and code changes here is low frequency

    b'\x1f\x8b' if sys.version_info[0] > 2 else '\x1f\x8b'